test/mocha/helpers.js

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as didKey from '@digitalbazaar/did-method-key';
+import {getCapabilitySigners, ZcapClient} from '@digitalbazaar/ezcap';
+import {Agent} from 'node:https';
+import {decodeSecretKeySeed} from 'bnid';
+import {Ed25519Signature2020} from '@digitalbazaar/ed25519-signature-2020';
+import {httpClient} from '@digitalbazaar/http-client';
+
+const agent = new Agent({rejectUnauthorized: false});
+const baseUrl = 'https://localhost:18443';
+const didKeyDriver = didKey.driver();
+
+const makeRequest = async ({url, json, headers, method}) => {
```

Comment:
Helpers should be at the bottom of the file. Ideally prefixed with `_`. Use `function` style instead of arrow.

Reply:
addressed here: https://github.com/digitalbazaar/bedrock-vc-exchanger/pull/10/commits/ddca903f15d3e9961a2b50a89ca4d3fd7ac6e1e8